modules/yardstick/src/main/java/org/apache/ignite/yardstick/IgniteBenchmarkArguments.java

```diff
@@ -279,6 +297,13 @@ public String configuration() {
     }
 
     /**
+     * @return File contains SQL queries.
+     */
+    public String queriesFile() {
```

Comment:
Rename to "loadTestQueriesFile" and improve Java doc referring to an identical comment above